operator/src/main/resources/Operator.properties

```diff
@@ -142,6 +142,8 @@ WLSKO-0191=Rolling restart of domain {0} completed
 WLSKO-0192=Executing make right domain operation, recheck count for server {0} is {1}.
 WLSKO-0193=Waiting for server {0} to start, recheck count is {1}.
 WLSKO-0194=Internal identity initialization step failed with exception {0}.
+WLSKO-0195=Successfully shutdown managed server {0} via REST interface.
+WLSKO-0196=Failed to shutdown managed server {0} via REST interface with response: {1}.
```

Comment:
I assume there's a pre-existing FINE that indicates start of shutdown for a managed server?
Perhaps all fines related to shutdown could be prefixed with something like "WL pod shutdown:" or some-such.
"managed server" --> "WebLogic server" (because admin server is technically not a managed server)

Reply:
When looking at all the messages, I didn't see any pre-existing FINE message that indicates the start of a shutdown of a managed server as we were basically deleting the pod which then called the stop scripts.  The only messages I found were in the stop-server.py script and look like the following:

print('Shutdown: Calling server shutdown with force = ' + force)
print('Shutdown: Successfully shutdown the server')
print('Shutdown: Exception in connect or shutdown')

I'm happy to change the message to have the prefix you suggested, as well as remove 'managed server'.

Reply:
Thanks for the review Tom.  I'll make the changes.

Reply:
Added another FINE log message for when we initiate shutdown of a server via REST.

Reply:
Looks good.